### DISCUSSION TITLE
[ideas] Add feature to set audit-settings of environment

### DISCUSSION THREAD

**Discussion by sanjeev40084** [microsoft/powerplatform-vscode#575] [Ideas]
Currently we have two different audit settings for production and non-prod environments. Although we can spin up new environment with pac cli, we still have to go to PPAC and configure to audit settings for the environment. It would be nice if settings gets added which allows setting audit settings during creation, update or reset of the environment.

**Comment by petrochuk** [comment]
We are adding ability to read any environment setting and are working on commands to modify them as well

![image](https://github.com/microsoft/powerplatform-vscode/assets/30735471/a15602a8-6cb5-4718-a5a2-cbef4079e578)

**Reply by sanjeev40084** [reply]
wow..really excited that you guys are already considering this. 👏

**Reply by sanjeev40084** [reply]
@petrochuk looks llike the june release (1.25.2) has added audit and lot's of other org settings functionality. I am trying to enable/disable audit settings for environment, but simple yes or no or 0 or 1 value doesn't seem to work, any idea what value should be passed to set/unset auditsettings. 

![image](https://github.com/microsoft/powerplatform-vscode/assets/3292092/954ea2a7-ae00-4cb4-867e-a51fa43e03a1)

**Reply by petrochuk** [reply]
true/false

I agree it is a little confusing at this time. We are looking at improving it.

**Reply by sanjeev40084** [reply]
@petrochuk  thank you that worked 👍

**Comment by sanjeev40084** [comment]
closing this discussion, since this feature is now available on current release ([June 2023, Version: 1.25.2](https://www.nuget.org/packages/Microsoft.PowerApps.CLI/1.25.2))

`pac org update-settings -n isauditenabled -v true`
